### ISSUE TITLE
README Update to Match New Docs

### ISSUE THREAD

**Issue by michaeltlombardi** [WahlNetwork/Vester#161]
The README is now out of date and points to the old, stale docs.

## Expected Behavior
```
AS A reader of the project
I WANT to have a link to the latest docs
SO THAT I can easily and quickly navigate to the documentation without having to remember the URL.
```
## Current Behavior
Clicking the link takes you to the ReadTheDocs stale documentation.

## Possible Solution
Update the README.

## Context
Users clicking the link are losing the new documentation available to them and, worse, going to _known-stale_ documentation.

Also, since the README is used for the intro page to the live docs, clicking the link there will actually take you away from the up-to-date documentation and put you back on the stale docs.

**Comment by brianbunke** [comment]
Yes, this should happen right away. See also latest comment in #70

**Comment by brianbunke** [comment]
Fixed in #164

**Comment by Midacts** [comment]
This issue should be closed, right?
The README.md points to the correct docs.
